### PR TITLE
Make `site-state-item` centered using `flex`

### PR DESCRIPTION
**Pull request by stevenjoezhang**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes was maked (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [ ] Docs in [NexT website](https://theme-next.org/docs/) have been added / updated (for new features).

## PR Type
**What kind of change does this PR introduce?**

- [ ] Bugfix.
- [ ] Feature.
- [x] Code style update (formatting, local variables).
- [x] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue resolved: N/A
屏幕快照 2019-03-17 上午2 14 52
屏幕快照 2019-03-17 上午2 14 57

## What is the new behavior?

- Screenshots with this changes: N/A
- Link to demo site with this changes: N/A
屏幕快照 2019-03-17 上午2 34 41

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by stevenjoezhang** [comment]
屏幕快照 2019-03-17 上午8 45 09
Yes, this should be a v7+ feature, some ancient browsers (about 4%) doesn't support `display: flex`

File: source/css/_common/components/sidebar/site-state.styl [hunk 1]
```diff
@@ -7,7 +9,6 @@
 }
 
 .site-state-item {
-  display: inline-block;
```
Comment (1v9) [inline source/css/_common/components/sidebar/site-state.styl]: [](https://i.loli.net/2019/03/17/5c8e2eae70937.gif)

Comment (stevenjoezhang) [reply source/css/_common/components/sidebar/site-state.styl]: @1v9 It's not about `display: inline-block`. I forgot to add `!important` to `display: flex` when `motion` enabled. I'll fix it

Comment (stevenjoezhang) [reply source/css/_common/components/sidebar/site-state.styl]: https://github.com/julianshapiro/velocity/issues/672

**Pull request merged**